src/main/java/com/snowflake/kafka/connector/internal/LoggerHandler.java

```diff
@@ -61,8 +73,8 @@ public static void setConnectGlobalInstanceId(String id) {
   public LoggerHandler(String name) {
     this.logger = LoggerFactory.getLogger(name);
 
-    META_LOGGER.info(
```

Comment:
reduced the log level from info to trace (and below from info to debug) since they were flooding the logs with not a loooot of usefulness

Reply:
👍